articles/ros2_threat_model.md

```diff
@@ -5088,7 +5081,7 @@ In the course of a first funded vulnerability assessment, <a style="color:black;
 [rvss]: https://arxiv.org/abs/1807.10357
 [safe_sec]: https://arxiv.org/abs/1806.06681
 [tb3_burger]: http://emanual.robotis.com/docs/en/platform/turtlebot3/specifications/#hardware-specifications
-[tb3_ros2_setup]:(http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/#setup
+[tb3_ros2_setup]:http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/#setup
```

Comment:
Is this okay? Or should be with space as the rest of the links?

Reply:
Good catch! We will fix it.

Reply:
Linter was throwing errors about the parenthesis, but yes, better to have the same spaces for all refs. Thanks!